docs/installation.rst

```diff
@@ -6,34 +6,41 @@ Prerequisites
 
 .. |driver link| replace:: **NVIDIA Driver**
 .. _driver link: https://www.nvidia.com/drivers
-.. |cuda link| replace:: **NVIDIA CUDA 10.0**
+.. |cuda link| replace:: **NVIDIA CUDA 11.0**
 .. _cuda link: https://developer.nvidia.com/cuda-downloads
-.. |mxnet link| replace:: **MXNet 1.3**
+.. |mxnet link| replace:: **MXNet**
 .. _mxnet link: http://mxnet.incubator.apache.org
-.. |pytorch link| replace:: **PyTorch 0.4**
+.. |pytorch link| replace:: **PyTorch**
 .. _pytorch link: https://pytorch.org
-.. |tf link| replace:: **TensorFlow 1.7**
+.. |tf link| replace:: **TensorFlow**
 .. _tf link: https://www.tensorflow.org
+.. |pddl link| replace:: **PaddlePaddle**
+.. _pddl link: https://www.paddlepaddle.org.cn
 .. |compatibility link| replace:: enhanced CUDA compatibility guide
 .. _compatibility link : https://docs.nvidia.com/deploy/cuda-compatibility/index.html#enhanced-compat-minor-releases
 
 1. Linux x64.
-2. |driver link|_ supporting `CUDA 10.0 <https://developer.nvidia.com/cuda-downloads>`__ or later (i.e., 410.48 or later driver releases).
-3. (Optional) One or more of the following deep learning frameworks:
+2. |driver link|_ supporting `CUDA 11.0 <https://developer.nvidia.com/cuda-downloads>`__ or later (i.e. 450.80.02 or later driver releases).
+3. [Optional] One or more of the following deep learning frameworks:
 
-  - |mxnet link|_ ``mxnet-cu100`` or later.
-  - |pytorch link|_ or later.
-  - |tf link|_ or later.
+  - |mxnet link|_
+  - |pytorch link|_
+  - |tf link|_
+  - |pddl link|_
 
 
 DALI in NGC Containers
 ----------------------
 
-DALI is preinstalled in the TensorFlow, PyTorch, and MXNet containers in versions 18.07 and
-later on `NVIDIA GPU Cloud <https://ngc.nvidia.com>`_.
+DALI is preinstalled in the `TensorFlow <https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorflow>`_,
```

Comment:
Please add the PaddlePaddle container here.

Reply:
done